broker/Cargo.toml

```diff
@@ -0,0 +1,39 @@
+[package]
+name = "neon_broker"
+version = "0.1.0"
+edition = "2021"
+
+[features]
+bench = []
+
+[[bin]]
+name = "neon_broker_bench"
+path = "src/bin/bench.rs"
+# build benchmarking binary only if explicitly requested with '--features bench'
+required-features = ["bench"]
```

Comment:
Checked out, I don't think it fits well the micro benchmarking format, for example it would be nice to print rps, not execution time. Also, #[bench] is unstable (yes, there is Criterion...). But I will add readme to the binary.

Reply:
Please revisit what `harness = false` does. It supports exactly this case. You write usual binary with main function that will be invoked with `cargo bench`. See this example from glommio https://github.com/DataDog/glommio/blob/master/glommio/Cargo.toml#L74

Reply:
Checked again, it seems impossible to run `cargo bench` without making it pass `--bench` to the binary, and clap doesn't support arguments with no value %)

Reply:
I was surely wrong, clap handles that, moved to [[bench]].